src/App.vue

```diff
@@ -2,36 +2,49 @@
 import NavigationBar from '@/components/NavigationBar.vue'
 </script>
 
+<script setup>
```

Comment:
Hi @salashhab  - I think two "script setup" sections won't work in one file. Did this work on your local environment?

Reply:
I did not test it in my environment. We did not do the handover of this part yet.

Reply:
Now I was able to test it in my local. Totally true, two script setup want work. I removed the first duplicated one. Now everything is working in my local.